http/pkger_http_server.go

```diff
@@ -36,10 +36,11 @@ func NewHandlerPkg(log *zap.Logger, errHandler influxdb.HTTPErrorHandler, svc pk
 	}
 
 	r := chi.NewRouter()
-	r.Use(middleware.RequestID)
-	r.Use(middleware.RealIP)
-	r.Use(traceMW)
```

Comment:
is pkger tracing being added somewhere else now?

Reply:
yah, its being done at the platform handler level. I never saw it before. Its really tucked away in there. It is duplicative with the trace mw that is already in place.

Reply:
Right on, just noticed that.

Reply:
it ws an oversight on my end during the initial pkger work